src/Utilities/Workspaces/SyntaxNodeExtensions.cs

```diff
@@ -24,6 +24,11 @@ internal static class SyntaxNodeExtensions
             }
         }
 
+        /// <summary>
+        /// Annotates a syntax node representing a type so that any missing imports get automatically added. Does not work in any other kinds of nodes.
+        /// </summary>
+        /// <param name="syntaxNode">The type node to annotate.</param>
+        /// <returns>The annotated type node.</returns>
         public static SyntaxNode WithAddImportsAnnotation(this SyntaxNode syntaxNode)
```

Comment:
@sharwell I documented this method we discussed yesterday, so people know it only works with Types. Let me know if you'd like to change the wording.

Reply:
I believe we should file a Roslyn bug here. Roslyn should throw an exception with an actionable message when an annotation is added to an unsupported node, instead of silently ignoring it.

Reply:
Done: https://github.com/dotnet/roslyn/issues/46481